mobile/library/cc/engine_builder.cc

```diff
@@ -498,11 +498,14 @@ std::string EngineBuilder::generateConfigStr() const {
   return config_str;
 }
 
+bool EngineBuilder::generateBootstrapAndCompare(std::string yaml) const {
+  return generatedStringMatchesGeneratedBoostrap(yaml, *generateBootstrap());
+}
+
 std::unique_ptr<envoy::config::bootstrap::v3::Bootstrap>
 EngineBuilder::generateBootstrapAndCompareForTests(std::string yaml) const {
   auto bootstrap = generateBootstrap();
```

Comment:
nit: This predates you, but can we use the explicit type here, per the style guide.

"The fundamental rule is: use type deduction only to make the code clearer or safer, and do not use it merely to avoid the inconvenience of writing an explicit type"
https://google.github.io/styleguide/cppguide.html#Type_deduction

Reply:
I guess it's a personal choice, but I find

```c++
auto bootstrap
```

clearer than

```c++
std::unique_ptr<envoy::config::bootstrap::v3::Bootstrap> bootrap
```

But I'll take your word for it :)

Reply:
Oh, for some reason I thought this was a string. auto is definitely shorter :)

Happy to leave this up to you, especially since this predates your PR.

Reply:
I updated it.